packages/deployments/contracts/contracts/core/connext/facets/RoutersFacet.sol

```diff
@@ -319,6 +344,33 @@ contract RoutersFacet is BaseConnextFacet {
     emit LiquidityFeeNumeratorUpdated(_numerator, msg.sender);
   }
 
+  /**
+   * @notice Allow router to use Portals
+   * @param _router - The router address to approve
+   */
+  function approveRouterForPortal(address _router) external {
```

Comment:
why is this not `onlyOwner`?

Is this intended to let any router approve themselves (in which case, this should be permissioned to onlyRouterOwner)?

Reply:
It should be `onlyOwner`!